packages/slice-machine/src/tracker.ts

```diff
@@ -63,7 +64,7 @@ export class SMTracker {
 
     return this.#client
       .then((client): void => {
-        void client.page(attributes);
+        void client.page(EventType.PageView, attributes);
```

Comment:
shouldn't it also track the page the was viewed rather than just saying "SliceMachine", or maybe it is something sent by the SDK automatically ?

Reply:
it is :)